resources/views/components/navbar/navbar.blade.php

```diff
@@ -1,187 +1,9 @@
-                                    >
-                                        <div class="flex flex-col pt-2 pb-2 w-full">
-                                            @foreach ($navItem['children'] as $menuItem)
-                                                <x-navbar.mobile.list-item
-                                                    :route="$menuItem['route'] ?? null"
-                                                    :params="$menuItem['params'] ?? null"
-                                                    :label="$menuItem['label']"
-                                                    :url="$menuItem['url'] ?? null"
-                                                />
-                                            @endforeach
-                                        </div>
-                                    </div>
-                                </div>
-                            @else
-                                <x-navbar.mobile.item
-                                    :route="$navItem['route']"
-                                    :params="$navItem['params'] ?? null"
-                                    :label="$navItem['label']"
-                                />
-                            @endif
-                        @endforeach
-
-                        <div class="py-4 dark:bg-black bg-theme-secondary-100">
-                            <div class="mx-6 space-y-3 divide-y divide-dashed divide-theme-secondary-300 dark:divide-theme-secondary-800">
-                                <x-navbar.mobile.setting-item title="{{ trans('general.select_theme') }}">
-                                    <livewire:navbar.dark-mode-toggle
-                                        active-icon="underline-moon"
-                                        inactive-icon="underline-sun"
-                                        setting="darkTheme"
-                                        mobile
-                                    />
-                                </x-navbar.mobile.setting-item>
-
-                                <x-navbar.mobile.setting-item
-                                    title="{{ trans('general.select_network') }}"
-                                    class="pt-3"
-                                >
-                                    <x-navbar.network-dropdown />
-                                </x-navbar.mobile.setting-item>
-
-                                <div class="flex pt-3 font-semibold dark:text-theme-secondary-500">
-                                    <livewire:price-ticker />
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </nav>
-    </div>
-</header>
+@props([
+    'navigation' => [],
+])
+
+<div class="z-30 md:sticky md:top-0">
+    <x-navbar.top />
+    <x-navbar.desktop :navigation="$navigation" />
+    <x-navbar.mobile :navigation="$navigation" />
```

Comment:
what was the reason for needing this?

Reply:
Just separated out the components as it was hard to differentiate between the two